app/services/schema_validator.rb

```diff
@@ -0,0 +1,43 @@
+require "json-schema"
+require "uri"
+
+class SchemaValidator < ApplicationService
```

Comment:
probably I should rename this, since this is to validate the CP API ...
maybe `CommonPlatformApiValidator`